### DISCUSSION TITLE
Daily Standup (Friday, May 27th)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#75] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
 ```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished auth middleware with @BrandonMagana 
 - Opened a PR for the same thing
 - Integrated Firebase SDK into the project

**What are your priorities for the next 24 hours?**:
 - Read more about error handling in express 
 - Fix the middleware's PR branch history

**Blockers**:
 - Been mentally tired lately

**Shoutouts**:
 - @everyone. Keep working hard babagees

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished basic Login view
 - Created aspirant API for login

**What are your priorities for the next 24 hours?**:
 - Start Signup view

**Blockers**:
 - None

**Shoutouts**:
 - @BrandonMagana @antonieto your code is great

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Started prototype for Search Aspirant view on Figma

**What are your priorities for the next 24 hours?**:
 - Finished it maybe

**Blockers**:
 - Figma stuff

**Shoutouts**:
 - @Eeveryone

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much really

**What are your priorities for the next 24 hours?**:
 - Work on signup flow

**Blockers**:
 - None
